backend-tests/tests/test_inventory.py

```diff
@@ -77,6 +82,7 @@ def tenants_users(clean_migrated_mongo_mt):
         username = "user@%s.com" % n
         password = "correcthorse"
         tenant = create_org(n, username, password)
+        tenants.append(tenant)
```

Comment:
So this was not working at all before? :exploding_head:

Reply:
it looks so...